errata.go

```diff
@@ -0,0 +1,51 @@
+package multihash
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"hash"
+)
+
+type identityMultihash struct {
+	bytes.Buffer
+}
+
+func (identityMultihash) BlockSize() int {
```

Comment:
bytes.Buffer is large, take this by pointer.

Reply:
Fixed.

Reply:
:coffee:

Reply:
(maybe go will just optimize this one away?)

Reply:
yeah if there's no variable name for the receiver at all I'm pretty sure there's no memory being shuffled.

(I made the other methods on this type take pointer receivers.)

(I kinda doubt it matters anyway, IME/ISTM that roughly three words is about where one starts seeing a difference, and that's about how big bytes.Buffer is.  But it probably depends on how you use it, I'm not really caring enough to be counting these nanos, etc, etc.)

Reply:
We should still make these consistent (I'm pretty sure tooling will complain, but the tooling is currently broken due to the build issues).